GoogleForCreators/web-stories-wp
# Template Updates: Beauty

## Context
Page layouts/templates have been updated to use provided animations.
_note_ [Also need to approve and merge asset update PR](https://github.com/google/web-stories-wp/pull/7170), the video shows the old asset but when that's approved and merged the image will be updated



## Summary
- Replace custom animations with editor provided animations according to [figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=7216%3A549493) and details in ticket
- Add a new SVG sticker (same as `beautyHeart` but without the shadow)
- Update images


## Relevant Technical Choices



## To-do
- [x] Additional PR to update image on page 8 (git lfs)
- [x] Show before/after in a video
- [x] Design updates


## User-facing changes
- Beauty template will no longer contain custom animations (animation dropdown will work properly on them)
- Beauty template will no longer have grid images, and will use stickers for the heart shape layers, one portrait changed and some copy on the same page (8)




https://user-images.githubusercontent.com/41136059/114610570-ebab7380-9c54-11eb-94bf-bf90c07e64ec.mp4


![Screen Shot 2021-04-09 at 6 03 15 PM](https://user-images.githubusercontent.com/41136059/114253761-6415ea00-9960-11eb-8ea3-bb24c346d1e5.png)

## Testing Instructions



### QA






This PR can be tested by following these steps:

1. On the dashboard, explore templates and choose to view the details on the Beauty template (Fresh & Bright)
2. On each page, observe that the animations and images have changed according to the specifications laid out in the ticket

### UAT








## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6969
